packages/neos-ui/src/manifest.dataloaders.js

```diff
@@ -192,8 +192,8 @@ manifest('main.dataloaders', {}, globalRegistry => {
             return Promise.all(
                 identifiers.map(identifier =>
                     this.resolveValue(options, identifier)
-                ).filter(promise => Boolean(promise)) // remove "null" values
-            );
+                )
+            ).then(results => [].concat(...results));
```

Comment:
Is this 100% part of the feature PR?

Reply:
Yes, otherwise this PR will not work and it was actually broken before.